api/views.py

```diff
@@ -62,19 +62,28 @@ def tasks_list_all(request: HttpRequest):
 
     count = None
     start = None
+    order = ["-created_at", "-id"]
 
     if "subject" in request.GET:
         filters["subject__abbr"] = request.GET["subject"]
     if "count" in request.GET:
         count = int(request.GET["count"])
     if "start" in request.GET:
         start = int(request.GET["start"])
+    if "sort" in request.GET:
```

Comment:
`if request.GET("sort") == "asc": ...`, we don't need the `else` branch, as it just sets the same default.